xbmc/windows/GUIWindowSplash.cpp

```diff
@@ -35,11 +36,8 @@ CGUIWindowSplash::~CGUIWindowSplash(void) = default;
 
 void CGUIWindowSplash::OnInitWindow()
 {
-  std::string splashImage = "special://home/media/Splash.png";
-  if (!XFILE::CFile::Exists(splashImage))
-    splashImage = "special://xbmc/media/Splash.png";
-
-  CLog::Log(LOGINFO, "load splash image: %s", CSpecialProtocol::TranslatePath(splashImage).c_str());
+  std::string splashImage = CUtil::GetSplashPath();
+  CLog::Log(LOGINFO, "load splash image: %s", splashImage.c_str());
```

Comment:
ahh, could you please remove this line .. I think it's a left-over or change the log level to debug.

Reply:
I would eliminate `splashImage` variable => `CLog::Log(LOGINFO, "load splash image: %s", CUtil::GetSplashPath().c_str());`

Or make the var at least `const`.

Reply:
@xhaggi it was `LOGINFO` before. So, why do you want to change that?

Reply:
The logging of the path makes no sense here. It's not logged in `CRenderSystemBase` so why should we do it here? I've introduced it in the initial PR so it should be fine if I say drop it ;)

Reply:
okay, I was just curious.

Reply:
removed